releasenotes/notes/remove-deprecated-factorizers-linear-solvers-4631870129749624.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
```

Comment:
This should be an upgrade note; the deprecation section is for documenting and marking a particular piece of functionality as something that is being removed in the future. We've already done that and this PR is following through on that earlier promise and is removing the functionality so it needs to be an upgrade note to document the removal as something that a user would potentially need to take manual intervention on when upgrading from a previous version.